sources/packages/api/src/route-controllers/institution-locations/institution-location.controller.service.ts

```diff
@@ -70,4 +75,34 @@ export class InstitutionLocationControllerService {
       ? DesignationStatus.Designated
       : DesignationStatus.NotDesignated;
   }
+
+  /**
+   * Get the Institution Location details.
+   * @param institutionId
+   * @param locationId
+   * @returns location details as InstitutionLocationFormAPIOutDTO.
+   */
+  async getInstitutionLocation(
+    institutionId: number,
```

Comment:
institutionId will be used only by the institution controller. so it can be optional here.